Tests/PrettyPrinterTests/IfStmtTests.swift

```diff
@@ -88,8 +88,8 @@ public class IfStmtTests: PrettyPrintTestCase {
 
       if var1 < var2 {
         let a = 23
-      } else if var3 <
-         var4 {
+      } else
+      if var3 < var4 {
```

Comment:
This seems like an odd place to break, and we may want to figure out how to handle this situation better (but we don't have to tackle it now in this PR).